internal/cmd/local/local_get_credentials.go

```diff
@@ -0,0 +1,80 @@
+package local
+
+import (
+	"fmt"
+	"github.com/airbytehq/abctl/internal/cmd/local/k8s"
+	"github.com/airbytehq/abctl/internal/cmd/local/localerr"
+	"github.com/airbytehq/abctl/internal/telemetry"
+	"github.com/pterm/pterm"
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+const (
+	airbyteAuthSecretName = "airbyte-auth-secrets"
+	airbyteNamespace      = "airbyte-abctl"
+)
+
+func NewCmdGetCredentials(provider k8s.Provider) *cobra.Command {
+	//spinner := &pterm.DefaultSpinner
+
+	cmd := &cobra.Command{
+		Use:   "get-credentials",
+		Short: "Get Airbyte user credentials",
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return nil
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return telClient.Wrap(cmd.Context(), telemetry.Install, func() error {
+
+				// TODO: We should check that we can actually talk to the cluster
+				client, err := defaultK8s(provider.Kubeconfig, provider.Context)
+				if err != nil {
+					return nil
+				}
+				secret, err := client.SecretGet(cmd.Context(), airbyteNamespace, airbyteAuthSecretName)
+				if err != nil {
+					return err
+				}
+
+				pterm.Success.Printfln(fmt.Sprintf("Getting your credentials: %s", secret.Name))
+				for k, v := range secret.Data {
+					if k == "jwt-signature-secret" {
+						continue
+					}
```

Comment:
Rather then being explicit about what we don't output, should we rather be explicit about what we _do_ output?